404.md

```diff
@@ -8,8 +8,7 @@ nav_exclude: true
 
 ## Oops, this isn't the page you're looking for. 
 
-Maybe our [homepage](https://opensearch.org/docs/latest)
-or one of the popular pages listed below can help. 
+Maybe our [home page](https://opensearch.org/docs/latest) or one the commonly visited pages below will help. If you need further support, please use the feedback feature on the right side of the screen to get in touch.
```

Comment:
```suggestion
Maybe our [home page](https://opensearch.org/docs/latest) or one of the commonly visited pages below will help. If you need further support, please use the feedback feature on the right side of the screen to get in touch.
```

Reply:
Just one omission here. Looks good to me. I'll let others weigh in if they'd like.